tools/generate-docs-integration-screenshot

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""Create or update a webhook integration screenshot using a test fixture."""
+
+# check for the venv
+from lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+import os
+import sys
+
+TOOLS_DIR = os.path.abspath(os.path.dirname(__file__))
+ROOT_DIR = os.path.dirname(TOOLS_DIR)
+
+sys.path.insert(0, ROOT_DIR)
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+import django
+
+django.setup()
+
+import argparse
+import subprocess
+from typing import Any, Dict, Tuple
+
+import cairosvg
+import requests
+import ujson
+
+from zerver.models import UserProfile, Message
+from zerver.lib.actions import do_create_user, notify_created_bot
+from zerver.lib.upload import upload_avatar_image
+from zerver.lib.actions import do_change_avatar_fields
+from zerver.lib.integrations import WebhookIntegration, INTEGRATIONS
+from zerver.lib.webhooks.common import get_fixture_http_headers
+
+def create_integration_bot(integration_name: str) -> UserProfile:
+    owner = UserProfile.objects.get(id=10)
+    bot_email = "{}-bot@example.com".format(integration_name)
+    bot_name = "{} Bot".format(integration_name.capitalize())
+    try:
+        bot = UserProfile.objects.get(email=bot_email)
+    except UserProfile.DoesNotExist:
+        bot = do_create_user(
+            email=bot_email,
+            password="123",
+            realm=owner.realm,
+            full_name=bot_name,
+            short_name=bot_name,
+            bot_type=UserProfile.INCOMING_WEBHOOK_BOT,
+            bot_owner=owner,
+        )
+        notify_created_bot(bot)
+
+        if integration.logo_url is None:
+            return bot
+        logo_path = os.path.join(ROOT_DIR, integration.logo_url.lstrip("/"))
+
+        if logo_path.endswith(".svg"):
+            logo_png_path = "/tmp/{}-logo.png".format(integration_name)
+            cairosvg.svg2png(url=logo_path, write_to=logo_png_path)
+            logo_path = logo_png_path
+
+        with open(logo_path, "rb") as f:
+            upload_avatar_image(f, owner, bot)
+            do_change_avatar_fields(bot, UserProfile.AVATAR_FROM_USER)
+
+    return bot
+
+def get_integration_fixture_names(path: str) -> Tuple[str, str]:
```

Comment:
I might call this `split_fixture_path` and put it in `zerver/lib/integrations.py`.